src/schema.rs

```diff
@@ -1,6 +1,7 @@
 use table::Table;
 
 #[derive(Debug, Clone, Copy)]
+/// A database schema.
```

Comment:
Add a few more words, like "This type is created by the `schema` function" with link to that function

(Same for `Table` and `Column` below)

Reply:
Done.

Reply:
Done.